src/app/content/highlights/components/SummaryPopup/FiltersList.tsx

```diff
@@ -0,0 +1,161 @@
+import { HighlightColorEnum } from '@openstax/highlighter/dist/api';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import styled, { css } from 'styled-components';
+import Times from '../../../../components/Times';
+import { textStyle } from '../../../../components/Typography';
+import { match, not } from '../../../../fpUtils';
+import theme from '../../../../theme';
+import { book as bookSelector } from '../../../selectors';
+import { LinkedArchiveTree, LinkedArchiveTreeSection } from '../../../types';
+import {
+  archiveTreeSectionIsBook,
+  archiveTreeSectionIsChapter,
+  flattenArchiveTree,
+} from '../../../utils/archiveTreeUtils';
+import { setChaptersFilter, setColorsFilter } from '../../actions';
+import { filtersChange } from '../../actions';
+import { chaptersFilter, colorsFilter } from '../../selectors';
+
+// tslint:disable-next-line: variable-name
+const RemoveIcon = styled.span`
+  padding: 0.5rem;
+  cursor: pointer;
+
+  svg {
+    height: 1rem;
+    width: 1rem;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+const ItemLabel = styled.span`
+  max-width: 80px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
this truncation is causing some weird spacing, the element with the truncated text appears to have more right margin than the others 

![image](https://user-images.githubusercontent.com/636227/70829852-c4bd0300-1dbc-11ea-8b0a-64eef4dffc4b.png)

Reply:
This is actually happening only for this `Function...` chapter. Probably because of different width of letters. I've could make like in cnx.org homepage for book excerptions. Let me know if you have something else in mind.
![image](https://user-images.githubusercontent.com/31478212/70910147-6cca0a80-200f-11ea-9f5e-4615afd45fa9.png)

Reply:
ok, if its just because of the letter width that makes sense, i'll look more closely with different filters